src/snowflakes/tests/test_searchv2.py

```diff
@@ -465,3 +465,54 @@ def test_collection_listing_db_view(workbook, testapp):
     assert r.json['@type'] == ['SnowflakeCollection', 'Collection']
     assert r.json['@context'] == '/terms/'
     assert r.json['description'] == 'Listing of Snowflakes'
+
+
+def test_auditv2_response(workbook, testapp):
```

Comment:
feel like the v2s should be removed from test function names.

Reply:
I've left the v2s to be consistent with the file name. This is something we can unwind when we delete old search code next round.